includes/stores/singlestore.php

```diff
@@ -149,19 +160,20 @@ function get_token( $args = array() ) {
 			'user_id' => get_current_user_id(),
 			'blog_id' => get_current_blog_id(),
 		);
-		$args = wp_parse_args( $args, $defaults );
+		$args     = wp_parse_args( $args, $defaults );
 
-		if ( !$args['id'] && !$args['service'] )
+		if ( ! $args['id'] && ! $args['service'] ) {
 			return false;
+		}
 
 		$post = get_post( $args['id'] );
 		if ( $post ) {
 			$meta = get_post_meta( $post->ID );
 			foreach ( $meta as $mid => $met ) {
-				$meta[$mid] = $met[0];
+				$meta[ $mid ] = $met[0];
 			}
 
-			$token_type = 'kr_request_token' == $post->post_type ? 'Keyring_Request_Token' : 'Keyring_Access_Token';
+			$token_type = 'kr_request_token' === $post->post_type ? 'Keyring_Request_Token' : 'Keyring_Access_Token';
```

Comment:
Should be fine, [post type is a string](https://developer.wordpress.org/reference/functions/get_post_type/#return).